packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -215,16 +215,24 @@
       </div>
       <div class="row">
         <div class="field-wrapper">
-          <a class="btn social-btn" :href="uris.github">
-            Continue With GitHub
-          </a>
+          <input
+            type="button"
+            id="continue-github"
+            value="CONTINUE WITH GITHUB"
```

Comment:
I think using normal sentence casing "Continue with GitHub" and using CSS to `text-transform: uppercase` will be more accessible and flexible